### PR TITLE
Tag services implementation

### PR DESCRIPTION
**Pull request by timreimherr**
This branch isn't finished yet, but I think I've switched between branches on commits and pushes too much. When I work on the solution while in another branch the 'TagServices.cs' file is missing. So once these changes are merged I'll focus on working in one branch at a time and then do a pull request so that everything is up to date.

Is that the 'best practice' way of using branches?

### CODE REVIEW

File: Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/CategoryServices.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GamingGuruBlog.Domain
+{
+    class CategoryServices
+    {
```
Comment (wojonet) [inline Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/CategoryServices.cs:10]: What's this class for?

File: Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/TagServices.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GamingGuruBlog.Domain
+{
+    class TagServices
+    {
```
Comment (wojonet) [inline Blog/GamingGuruBlog.Web/GamingGuruBlog.Domain/TagServices.cs]: Or this class?

**Comment by timreimherr** [comment]
Oh, these classes are to break up the BlogServices and keep it from being
so big. Like we discussed,right? To keep the entities separate rather than
one services class handling all the entities.I'm going to put all the
category methods that are currently in the Blogpost services class into the
CategoryServices class and the same for the tags into TagServices class.
Then, BlogServices will take an instance of each for it's category and tag
needs.

Am I thinking of this correctly?

Tim

**Comment by wojonet** [comment]
Yeah but then why did you create the pull request if you weren't done?

**Comment by timreimherr** [comment]
Because I realized making changes in the solution and committing them in a
different branch was keeping them from being available in the other branch.
Does that make sense? So, requested the pull request so those changes would
persist through the branches. Now I think I get it, I probably just need to
roll those back, finish a branch and then do the merge request. Jumping
around and committing changes in different branches isn't the best way to
do it.

So, should I just do that?
Roll them back and then stick in one branch until it's done?

**Comment by timreimherr** [comment]
@wojonet 
Okay, I made some more commits to this branch and I believe it's finished. Do I need to and can I make another pull request for this branch?

**Pull request merged**